coffeine/power_features.py

```diff
@@ -53,6 +55,214 @@ def _compute_cospectral_covs(epochs, n_fft, n_overlap, fmin, fmax, fs):
+    subset : list-like
+        A selection of valid keys to return a subset of frequency
+        bands from a collection.
+
+    Returns
+    -------
+    frequency_bands : dict
+        The band definitions.
+    
+    References
+    ----------
+    [1] Larson-Prior, L. J., R. Oostenveld, S. Della Penna, G. Michalareas,
+        F. Prior, A. Babajani-Feremi, J-M Schoffelen, et al. 2013.
+        “Adding Dynamics to the Human Connectome Project with MEG.”
+        NeuroImage 80 (October): 190–201.
+    [2] D. Sabbagh, P. Ablin, G. Varoquaux, A. Gramfort, and D.A. Engemann.
+        Predictive regression modeling with MEG/EEG: from source power
+        to signals and cognitive states.
+        *NeuroImage*, page 116893,2020. ISSN 1053-8119.
+        https://doi.org/10.1016/j.neuroimage.2020.116893
+    [3] D. A. Engemann, O. Kozynets, D. Sabbagh, G. Lemaître, G. Varoquaux,
+        F. Liem, and A. Gramfort Combining magnetoencephalography with
+        magnetic resonance imaging enhances learning of surrogate-biomarkers.
+        eLife, 9:e54055, 2020 <https://elifesciences.org/articles/54055>
+    [4] Jobert, M., Wilson, F.J., Ruigt, G.S., Brunovsky, M., Prichep,
+        L.S., Drinkenburg, W.H. and IPEG Pharmaco-EEG Guideline Committee,
+        2012. Guidelines for the recording and evaluation of pharmaco-EEG data
+        in man: the International Pharmaco-EEG Society (IPEG).
+        Neuropsychobiology, 66(4), pp.201-220.
+    """
+    frequency_bands = dict()
+    if collection == 'ipeg':
+        frequency_bands.update({
+            "delta": (1.5, 6.0),
+            "theta": (6.0, 8.5),
+            "alpha1": (8.5, 10.5),
+            "alpha2": (10.5, 12.5),
+            "beta1": (12.5, 18.5),
+            "beta2": (18.5, 21.0),
+            "beta3": (21.0, 30.0),
+            "gamma": (30.0, 40.0),
+        })  # total: 1.5-30; dominant: 6-12.5
+    if collection == 'ipeg_aggregated':
+        frequency_bands.update({
+            'total': (1.5, 30),
+            'dominant': (6, 12.5)
+        })
+    elif collection == 'hcp':
+        # https://www.humanconnectome.org/storage/app/media/documentation/
+        # s500/hcps500meg2releasereferencemanual.pdf
+        frequency_bands.update({    
+            'low': (0.1, 1.5),  # added later in [2,3].
+            'delta': (1.5, 4.0),
+            'theta': (4.0, 8.0),
+            'alpha': (8.0, 15.0),
+            'beta_low': (15.0, 26.0),
+            'beta_high': (26.0, 35.0),
+            'gamma_low': (35.0, 50.0),
+            'gamma_mid': (50.0, 76.0),
+            'gamma_high': (76.0, 120.0)
+        })
+    elif collection == 'hcp_aggregated':
+        frequency_bands.update({
+            'wide_band': (1.5, 150.0)
+        })
+    if subset is not None:
+        frequency_bands = {
+            band: freqs for band, freqs in frequency_bands.items()
+            if band in subset
+        }
+    return frequency_bands
+
+
+def make_coffeine_df(C: np.ndarray,
```

Comment:
@apmellot @antoinecollas @apmellot @DavidSabbagh @hubertjb @bmalezieux 

A small but important & overdue function :) What do you think about its naming? I think I am in principle happy with it. Other intuitive options would be to call it `make_coffeine_data_frame` or `make_covariance_data_frame`.  I would be OK with making it explicit (`data_frame` rather than `df`). Slight preference for coffeine `make_coffeine_data_frame` for branding and energizing weirdness. I understand `make_coffeine_data_frame`  could be more boring and predictable (in the dispassionate scientific sense), which would be not bad either.

Any thoughts? (for the bigger picture, please take minute to zoom out and look at the entire PR / the linked issues).

Reply:
I prefer `make_coffeine_data_frame` to be consistent with the function `make_coffeine`